src/redux/reducers.js

```diff
@@ -0,0 +1,30 @@
+// Initial State
+const initialState = {
+    count: 0,
+    updated: 'Never'
+};
+
+//Helpers
+function setUpdated() {
+    var now = Date.now();
+    var date = new Date(now);
+    var dateTime = date.toLocaleString();
+    return dateTime;
+}
+
+const countReducer = (state = initialState, action) => {
+    switch (action.type) {
+        case 'INCREMENT_COUNT': {
+            return {
+                ...state,
+                count: state.count + 1,
+                updated: setUpdated()
```

Comment:
Ooh! I'm torn on this one. Should the action or the reducer set this timestamp? I think it would be an interesting exercise to write a test for this reducer

Reply:
I've left it in the reducer for now, unit test for the reducer can be found in reducers.test.js file and run with 'npm run test reducers.test.js'.

I've bundled tests into a test file at root level - is this best practice or would it be better to have each test file nested with it's component?